components/year_slider.py

```diff
@@ -0,0 +1,67 @@
+from dash import dcc, html
+import dash_bootstrap_components as dbc
+
+
+def slider(id, container_id):
+    return html.Div(
+        id=container_id,
+        children=[
+            dcc.Slider(
+                id=id,
+                min=0,
+                max=0,
+                value=None,
+                step=None,
+                included=False,
+            ),
+        ],
+    )
+
+
+## Helper function to create the slider configuration
+# @param expenditure_years: list of years from the expenditure dataset
+# @param outcome_years: list of years from the outcome dataset
+# @return: slider configuration dictionary with the following
+#           - style: style configuration for the slider
+#           - marks: marks configuration for the slider
+#           - selected_year: the selected year
+#           - min_year: the minimum year
+#           - max_year: the maximum year
+#           - tooltip: tooltip configuration for the slider
+def slider_helper(expenditure_years, outcome_years):
```

Comment:
Consider calling it `get_slider_config` to indicate the intent of the function? `slider_helper` doesn't say what it is helping with.